### ISSUE TITLE
Can somebody teach me where to place the bf holding gf-dead.png?

### ISSUE THREAD

**Issue by Ralphy999** [luckydog7/Funkin-android#576]
If you know could you help me?

**Comment by DanisLixo** [comment]
BRUHHHH, this is so coincidence, I was doing a Alt xml file for bf_Holding_GF and bf_Holding_GF--DEAD

**Comment by DanisLixo** [comment]
and, it actually works
![Screenshot_20210501-002229](https://user-images.githubusercontent.com/58278276/116769600-ac7e7000-aa13-11eb-9b82-39f267fb132f.png)

**Comment by DanisLixo** [comment]
_ignore miku_

**Comment by DanisLixo** [comment]
yeah
![Screenshot_20210501-004727](https://user-images.githubusercontent.com/58278276/116770035-ffa5f200-aa16-11eb-9705-556b93d723a0.png)
he is dead

**Comment by MrTks216** [comment]
> yeah
> ![Screenshot_20210501-004727](https://user-images.githubusercontent.com/58278276/116770035-ffa5f200-aa16-11eb-9705-556b93d723a0.png)
> he is dead

damn that cool, can i have it?

**Comment by DanisLixo** [comment]
> > yeah
> > ![Screenshot_20210501-004727](https://user-images.githubusercontent.com/58278276/116770035-ffa5f200-aa16-11eb-9705-556b93d723a0.png)
> > he is dead
> 
> damn that cool, can i have it?

yes, but first I need to fix the crash, probably tomorrow I send!

**Comment by MrTks216** [comment]
> > > yeah
> > > ![Screenshot_20210501-004727](https://user-images.githubusercontent.com/58278276/116770035-ffa5f200-aa16-11eb-9705-556b93d723a0.png)
> > > he is dead
> > 
> > damn that cool, can i have it?
> 
> yes, but first I need to fix the crash, probably tomorrow I send!

Alright, thx

**Comment by Ralphy999** [comment]
> > > yeah
> > > ![Screenshot_20210501-004727](https://user-images.githubusercontent.com/58278276/116770035-ffa5f200-aa16-11eb-9705-556b93d723a0.png)
> > > he is dead
> > 
> > 
> > damn that cool, can i have it?
> 
> yes, but first I need to fix the crash, probably tomorrow I send!

Thx bro
